package.json

```diff
@@ -3,14 +3,14 @@
   "version": "1.0.0-beta.1",
   "description": "",
   "dependencies": {
-    "@1hive/apps-token-manager": "3.0.0",
+    "@1hive/apps-token-manager": "3.0.1",
```

Comment:
Is there a branch with this new version? I guess this new version only exposes the `getTokenManager` function?

Reply:
Yeah exactly, it is on this branch: https://github.com/1Hive/token-manager-app/tree/latest-aragon-os which I was waiting until the aragon OS is out of release candidate stage until I merged. Although I probably could just merge it, just concerned it might mess up anyone looking at it when using the gardens template or old conviction voting which uses the old aragon OS is all.